binding-service-impl/src/main/java/io/mosip/esignet/services/PublicKeyRegistryServiceImpl.java

```diff
@@ -12,6 +12,8 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.time.LocalDate;
```

Comment:
is LocalDate being used, else let's remove the import

Reply:
removed